### PR TITLE
beneficialOwnershipStatement -> ownershipOrControlStatement

### PR DESCRIPTION
**Pull request by timgdavies**
In reviewing training materials around BODS, and exploring different ways the standard may be used to represent data, we identified cases where a source may relate an `entity` and `interestedParty` but without wishing to make an explicit claim that the relationship is one of Beneficial Ownership.

As  a result of this, and as raised in #99, we believe that the beneficial ownership statement should be renamed `ownershipOrControlStatement`.

### CODE REVIEW

File: examples/.pytest_cache/v/cache/nodeids [hunk 1]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (ScatteredInk) [inline examples/.pytest_cache/v/cache/nodeids]: I don't think we should be tracking these - .pytest_cache should be in our .gitignore

File: tests/.pytest_cache/v/cache/lastfailed [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (ScatteredInk) [inline tests/.pytest_cache/v/cache/lastfailed]: I don't think we should be tracking these - .pytest_cache should be in our .gitignore

File: tests/.pytest_cache/v/cache/nodeids [hunk 3]
```diff
@@ -0,0 +1,43 @@
+[
```
Comment (ScatteredInk) [inline tests/.pytest_cache/v/cache/nodeids]: I don't think we should be tracking these - .pytest_cache should be in our .gitignore

File: examples/draft.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"statementGroups":[{},{},{},{}]}
```
Comment (ScatteredInk) [inline examples/draft.json]: Everything at top-level in /examples uses the old statementGroups model and can be removed.

File: docs/schema/modelling-beneficial-ownership-info.md [hunk 5]
```diff
@@ -12,17 +12,17 @@ The concept of a 'statement' is at the heart of the Beneficial Ownership Data St
...
 
-![A Beneficial ownership statement block containing two 'interests': one a 60% share-holding interest, the other a 30% voting-rights interest](_assets/data-schema-model-1.svg)
+![A ownership or control statement block containing two 'interests': one a 60% share-holding interest, the other a 30% voting-rights interest](_assets/data-schema-model-1.svg)
 
-Details of the subject of a beneficial ownership statement and its interested party are wrapped in their own statements. The beneficial ownership statement refers out to these statements, acting as a connector.
+Details of the subject of an ownership or control statement and its interested party are wrapped in their own statements. The ownership or control statement refers out to these statements, acting as a connector.
 
-![A Beneficial ownership statement block containing two 'interests': one a 60% share-holding interest, the other a 30% voting-rights interest](_assets/data-schema-model-2.svg)
+![An ownership or control ownership statement block containing two 'interests': one a 60% share-holding interest, the other a 30% voting-rights interest](_assets/data-schema-model-2.svg)
```
Comment (ScatteredInk) [inline docs/schema/modelling-beneficial-ownership-info.md]: "An ownership or control ownership statement block" should be "An ownership or control statement block"

File: docs/schema/modelling-beneficial-ownership-info.md [hunk 6]
```diff
@@ -12,17 +12,17 @@ The concept of a 'statement' is at the heart of the Beneficial Ownership Data St
...
 
 ## Statements as claims
 
 Each statement represents a claim about beneficial ownership made by a particular source at a particular point in time.
 
-![A Beneficial ownership statement block containing a source block with type-selfDeclaration, retrievedAt date of 2018-11-07 and assertedBy value of Acme Industries Ltd. Statement also has statementDate of 2018-07-12](_assets/data-schema-model-3.svg)
+![An ownership or control ownership statement block containing a source block with type-selfDeclaration, retrievedAt date of 2018-11-07 and assertedBy value of Acme Industries Ltd. Statement also has statementDate of 2018-07-12](_assets/data-schema-model-3.svg)
```
Comment (ScatteredInk) [inline docs/schema/modelling-beneficial-ownership-info.md]: "An ownership or control ownership statement block" should be "An ownership or control statement block"

File: docs/schema/data-schema-reference.md [hunk 7]
```diff
@@ -68,18 +68,18 @@ DETAILS OF STATEMENT IDENTIFIERS HERE
...
 
 Common components: 
 
 <!--TODO - CHECK THIS -->Statements must be packaged up in a [statement package](#statement-packages) for a [data release](building-data-releases.md).
 
-### BeneficialOwnershipStatement
+### OwnershipOrControlStatement
 
-A beneficial ownership statement is made up of statements about an entity, an interestedParty (either an entity, a person or null party), and details of the interest. Additionally, annotations on the interest, provenance and versioning information can be provided.
+An ownership or control statement is made up of statements about an entity, an interestedParty (either an entity, a person or null party), and details of the interest. Additionally, annotations on the interest, provenance and versioning information can be provided.
```
Comment (ScatteredInk) [inline docs/schema/data-schema-reference.md]: 'unspecified' rather than null party

**Comment by timgdavies** [comment]
Thanks @ScatteredInk  - I've made these changes and will merge shortly once checks complete.

**Pull request merged**